shop_richcatalog/templates/catalog/catalog_grid_plugin.html

```diff
@@ -1,21 +1,8 @@
-{% load thumbnail %}
+{% load cms_tags sekizai_tags %}
 {% load static from staticfiles %}
-{% for catalog in catalog_list %}
-    {% if forloop.first %}<div class="row">{% endif %}
-    <div class="col-xs-12 col-sm-6 col-md-3">
-        <div class="thumbnail">
-            {% if catalog.image %}
-                <img src="{% thumbnail catalog.image 400x200 upscale %}" alt="{{ catalog.get_name }}" />
-            {% else %}
-                <img src="{% static 'js/holder.js/400x200' %}" alt="{{ catalog.get_name }}" />
-            {% endif %}
-            <div class="caption">
-                <p class="catalog_description">{{ catalog.description }}</p>
-                <p><a href="{{ catalog.get_absolute_url }}"
-                      class="btn btn-primary btn-sm" role="button">View Products</a></p>
-            </div>
-        </div>
-    </div>
-    {% if forloop.counter|divisibleby:4 %}</div><div class="row">{% endif %}
-    {% if forloop.last %}</div>{% endif %}
-{% endfor %}
+{% include "catalog/catalog_grid.html" with catalog_list=plugin_catalog_list %}
+{% include "product/product_grid.html" with object_list=plugin_product_list %}
+{% addtoblock 'js' %}
+  <script type="text/javascript" src="{% static 'js/equalsizer.js' %}"></script>
```

Comment:
This app doesn't come bundled with equalsizer.js in its static content, and I don't think it should either. The templates for reusable Django apps shouldn't make use of any particular styling like Bootstrap, and should only include static content which it needs to function properly; these templates should be very minimal. If we want to use equalsizer.js here, then this template should be overridden in our projects.